examples/test_server.rs

```diff
@@ -47,14 +48,24 @@ impl Authorizer for MockUser {
     fn password(&mut self) -> PasswordFuture {
         Box::pin(async move { Password::from_known(self.password.clone()) })
     }
+
+    #[inline]
+    fn setup(&mut self, data_exists: bool) -> SetupFuture {
+        let new_mnemonic = Mnemonic::sample(&mut OsRng);
+        if data_exists {
+            Box::pin(async move { Setup::Login })
+        } else {
+            Box::pin(async move { Setup::CreateAccount(new_mnemonic) })
+        }
+    }
 }
 
 #[async_std::main]
 async fn main() -> Result<(), Error> {
     let test_dir = tempfile::tempdir()?;
     let mut config =
         Config::try_default().expect("Unable to generate the default server configuration.");
-    config.data_path = test_dir.path().join("storage.dat");
+    config.data_path[Network::Dolphin] = test_dir.path().join("storage-dolphin.dat");
```

Comment:
What about the rest of the files? Don't leave them stranding.

Reply:
added modifications to config to support the other network files